src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32Program.cs

```diff
@@ -403,65 +395,62 @@ private static Win32Program InternetShortcutProgram(string path)
                     {
                         urlPath = line.Substring(urlPrefix.Length);
 
-                        try
+                        if (!Uri.TryCreate(urlPath, UriKind.RelativeOrAbsolute, out Uri _))
```

Comment:
Use "Safe" TryParse (which does not throw an exception, but we log the warning anyway for debugging purpose